### ISSUE TITLE
[Bug] rdp keyboard layouts are wrong, needs solving as per Guacamole faq.

### ISSUE THREAD

**Issue by Zixim** [gnmyt/Nexterm#112]
### General

- [X] I have updated to the latest version of Nexterm.
- [X] My bug has not been reported yet.

### The Bug

My pc has Belgian keyboard layout (which is one of the azerty layouts), the rdp client also has Belgian layout.
RDP into the client and the keys in the client are qwerty.

This is a known 'issue' when using the guacamole backend, see here : 
https://guacamole.apache.org/faq/#does-guacamole-support-my-keyboard-layout

Here is how I solve this in guacamole :
![image](https://github.com/user-attachments/assets/1251baaa-78ae-444f-affc-49b837056ab8)

Basically all non-qwerty layouts will have this issue.

### What device are you using to access the page?

In the browser

### Which operating system is your Nexterm instance running on?

Linux

**Comment by gnmyt** [comment]
Thanks for reporting this. I've opened PR #374 that uses all keyboard layouts that guacamole supports.
